CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 # SMESH cmake build file
```

Comment:
@trelau Is increasing the cmake_minimum version to 3.3 ok? This would make packaging for conda a bit easier.

Reply:
This is probably fine